### DISCUSSION TITLE
0.4.0

### DISCUSSION THREAD

**Discussion by zu1k** [zu1k/good-mitm#51] [Announcements]
## What's Changed
* Support transparent proxy by @zu1k in https://github.com/zu1k/good-mitm/pull/50

## New Contributors
* @zu1k made their first contribution in https://github.com/zu1k/good-mitm/pull/50

**Full Changelog**: https://github.com/zu1k/good-mitm/compare/0.3.6...0.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/zu1k/good-mitm/releases/tag/0.4.0'>0.4.0</a>.</em>

**Comment by embeddedlove** [comment]
支持v6透明吗
